.github/workflows/remind-docs-and-tests.yml

```diff
@@ -1,6 +1,6 @@
 name: Remind docs and tests
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
Did this along the way - using `pull_request_target` here since this workflow uses a secret, that will allow it to work for contributor PRs.